### PR TITLE
[Doc] PR for Improve Configure Yugabyte Platform docs and Remove "Next step" sections (Doc11014)

### PR DESCRIPTION
**Pull request by lizayugabyte**

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ showAsideToc: true
 
 </ul>
 
-You can configure the on-premises cloud provider for YugabyteDB using the Yugabyte Platform UI. If no cloud providers are configured, the main Dashboard prompts you to configure at least one cloud provider.
+You can configure the on-premises cloud provider for YugabyteDB using the Yugabyte Platform UI. If no cloud providers are configured, the main Dashboard prompts you to configure at least one cloud provider, as per the following illustration:
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:68]: Yugabyte Platform UI - changed to console elsewhere

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 2]
```diff
@@ -153,74 +153,68 @@ For mount paths, use a mount point with enough space to contain your node densit
...
 
-After finishing the cloud provider configuration, click Manage Instances to provision as many nodes as your application requires.
+After finishing the cloud provider configuration, click **Manage Instances** to provision as many nodes as your application requires.
 
 ### Add nodes
 
 ![Configure On-Premises Cloud Provider](/images/ee/onprem/configure-onprem-4.png)
 
-For each node you want to add, click Add Instances to add a YugabyteDB node. You can use DNS names or IP addresses when adding instances. (Instance ID is an optional user-defined identifier.)
+For each node you want to add, click **Add Instances** to add a YugabyteDB node. You can use DNS names or IP addresses when adding instances (instance ID is an optional user-defined identifier.)
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:168]: identifier).

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 3]
```diff
@@ -153,74 +153,68 @@ For mount paths, use a mount point with enough space to contain your node densit
...
 1. Copy and paste the Python script prompted in the UI and substitute for a node IP address and mount points. 
-
-    \
-    (Optional) Use the `--ask_password` flag if the sudo user requires password authentication.
-
+Optionally, use the `--ask_password` flag if the sudo user requires password authentication.
+    
     ```output
-    bash-4.4# /opt/yugabyte/yugaware/data/provision/9cf26f3b-4c7c-451a-880d-593f2f76efce/provision_instance.py --ip 10.9.116.65 --mount_points /data --ask_password
+bash-4.4# /opt/yugabyte/yugaware/data/provision/9cf26f3b-4c7c-451a-880d-593f2f76efce/provision_instance.py --ip 10.9.116.65 --mount_points /data --ask_password
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:200]: Check preview - looks like this needs to be indented

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 4]
```diff
@@ -258,26 +252,26 @@ This table is based on the information on the [default ports page](/latest/refer
 
 ##### Pre-provision nodes manually
 
-This process carries out all provisioning tasks on the database nodes which require elevated privileges. Once the database nodes have been prepared in this way, the Universe creation process from the Platform server will connect with the nodes only via the `yugabyte` user, and not require any elevation of privileges to deploy and operate the YugabyteDB Universe.
+This process carries out all provisioning tasks on the database nodes which require elevated privileges. Once the database nodes have been prepared in this way, the universe creation process from the Yugabyte Platform server will connect with the nodes only via the `yugabyte` user, and not require any elevation of privileges to deploy and operate the YugabyteDB universe.
 
 Physical nodes (or cloud instances) are installed with a standard Centos 7 server image. The following steps are to be followed on each physical node, prior to universe creation:
 
 1. Log into each database node as a user with sudo enabled (the “centos” user in centos7 images)
```
Comment (ddhodge) [inline docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:259]: period

**Review by ddhodge — APPROVED** [review]
Minor comments

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md [hunk 5]
```diff
@@ -68,94 +68,88 @@ showAsideToc: true
...
 
-- Directly provide your [AWS Access Key ID and Secret Key](http://docs.aws.amazon.com/general/latest/gr/managing-aws-access-keys.html)
-- Attach an [IAM role](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html) to the Yugabyte Platform VM in the **EC2** tab. For more information, see [Deploy the YugabyteDB universe using an IAM role](../../../install-yugabyte-platform/prepare-environment/aws/#deploy-the-yugabytedb-universe-using-an-iam-role) . 
+### SSH Key Pairs
 
-## SSH Key Pairs
+In order to be able to provision EC2 instances with YugabyteDB, Yugabyte Platform requires SSH access. The following are two ways to provide SSH access:
 
-In order to be able to provision EC2 instances with YugabyteDB, the Yugabyte Platform requires SSH access. Here are two ways to provide SSH access:
+- Enable Yugabyte Platform to create and manage [Key Pairs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html). In this mode, Yugabyte Platform creates SSH Key Pairs across all the regions you choose to set up and store the relevant private key part of these locally in order to SSH into future EC2 instances.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/aws.md:100]: yes

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/gcp.md [hunk 6]
```diff
@@ -65,49 +65,43 @@ showAsideToc: true
...
+  Note that choosing to use the same VPC as Yugabyte Platform is an advanced option, which assumes that you are in complete control over this VPC and will be responsible for setting up the networking, SSH access, and firewall rules for it.
 
-- Finally, click **Save** and give it a couple of minutes, as it will need to do a bit of work in the background. This includes generating a new VPC, a network, subnetworks in all available regions, as well as a new firewall rule, VPC peering for network connectivity and a custom SSH key pair for Yugabyte Platform-to-YugabyteDB connectivity.
 
-Note: Choosing to use the same VPC as Yugabyte Platform is an advanced option, which currently assumes that you are in complete control over this VPC and will be responsible for setting up the networking, SSH access, and firewall rules for it!
+Click **Save** and wait a few minutes for the configuration to complete. This includes generating a new VPC, a network, subnetworks in all available regions, as well as a new firewall rule, VPC peering for network connectivity and a custom SSH key pair for Yugabyte Platform-to-YugabyteDB connectivity.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/gcp.md:97]: fixed

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:68]: Changed

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:168]: fixed

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:200]: fixed

File: docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:259]: fixed

**Pull request merged**